frontend/src/components/details/recommendations.svelte

```diff
@@ -7,15 +7,15 @@
     export let mediaType: string
 </script>
 
-{#if recommendations.length}
+{#if recommendations.length >= 5}
```

Comment:
Does this mean movies/series with less won't have recommendations?

Reply:
Yes, i feel like every movie/Series MOSTLY has more than 5, and only very few dont. Mostly new movie/Series dont have it to start with but get over 5 really fast.

Reply:
Do we not like having recommendations shown when there are only 1-4? Feels like a bit of a weird "fix" to have a static number like that